include/verilated_vpi.cpp

```diff
@@ -2384,122 +2422,210 @@ static void vl_strprintf(std::string& buffer, char const* fmt, ...) {
     va_end(args);
 }
 
-void vl_get_value(const VerilatedVar* varp, void* varDatap, p_vpi_value valuep,
-                  const char* fullname) {
+template<typename T>
+struct VarAccessInfo final {
+    size_t bitOffset;
+    size_t wordOffset;
+    T mask_lo, mask_hi;
+    T* ptr;
+};
+
+template<typename T>
+VarAccessInfo<T> vl_var_access_info(const VerilatedVpioVarBase* vop, size_t bitCount, size_t addOffset) {
+    size_t wordBits = sizeof(T) * 8;
```

Comment:
```suggestion
    const size_t wordBits = sizeof(T) * 8;
```

Check everywhere for more consts.